nbformat/tests/test_sign.py

```diff
@@ -4,9 +4,11 @@
 # Distributed under the terms of the Modified BSD License.
 
 import copy
+import os
 import shutil
 import time
 import tempfile
+import testpath
```

Comment:
It appears that this is causing the CI to fail - if we don't use a requirements.txt, how can we ensure that it's included in the packaging?